### PR TITLE
Updated API configuration to complement scraper

### PR DESCRIPTION
**Pull request by zaidokal**
# JIRA Ticket Link

https://skaarkzo-capstone.atlassian.net/browse/SUSTAIN-246

# Summary of Changes

Updated API configuration to complement scraper

### CODE REVIEW

File: src/app/services/apiService.ts [hunk 1]
```diff
@@ -13,15 +13,19 @@ const isError = (error: unknown): error is Error => {
 export const postCompanySearch = async (
   companyName: string
 ): Promise<Company[]> => {
-  const apiUrl = "http://localhost:8000/api/main/search/";
```
Comment (skhan943) [inline src/app/services/apiService.ts:16]: I think we should move the URL to a config file or something, it might end up being used in many places

**Review by skhan943 — CHANGES_REQUESTED** [review]
Generally LGTM, few changes

File: src/app/services/apiService.ts [hunk 2]
```diff
@@ -13,15 +13,19 @@ const isError = (error: unknown): error is Error => {
...
+  const apiUrl = "http://localhost:8000/api/main/company";
 
   try {
     const response = await fetch(apiUrl, {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
-      body: JSON.stringify({ companyName }),
```
Comment (skhan943) [inline src/app/services/apiService.ts:24]: standardize company_name

File: src/app/services/apiService.ts [hunk 3]
```diff
@@ -13,15 +13,19 @@ const isError = (error: unknown): error is Error => {
...
+      body: JSON.stringify({
+        company_name: companyName,
```
Comment (skhan943) [inline src/app/services/apiService.ts:25]: Here too

File: src/app/hooks/useFetchEvaluatedCompanies.ts [hunk 4]
```diff
@@ -13,20 +13,20 @@ const useFetchEvaluatedCompanies = (handleError: (context: string, error: unknow
...
-      } finally {
-        setIsLoading(false);
-      }
-    };
+  const loadCompanies = async () => {
+    try {
+      const data = await fetchEvaluatedCompanies();
+      setCompanies(data);
+    } catch (error) {
+      handleError("Backend is not responding. Please try again later", error);
```
[the remark this answers is not in the document]
Comment (zaidokal) [reply src/app/hooks/useFetchEvaluatedCompanies.ts]: Done

File: src/app/services/apiService.ts (hunk 1 above)
Comment (zaidokal) [reply src/app/services/apiService.ts:16]: Good idea, done!

File: src/app/services/apiService.ts (hunk 2 above)
Comment (zaidokal) [reply src/app/services/apiService.ts:24]: Nice catch! Done!

File: src/app/services/apiService.ts (hunk 3 above)
Comment (zaidokal) [reply src/app/services/apiService.ts]: Done

**Review by skhan943 — APPROVED** [review]
LGTM

**Review by rakhshankhan — APPROVED** [review]

**Pull request merged**